pkg/api/v1/conversion_generated.go

```diff
@@ -23,6 +23,7 @@ import (
 
 	api "k8s.io/kubernetes/pkg/api"
 	resource "k8s.io/kubernetes/pkg/api/resource"
+	experimental "k8s.io/kubernetes/pkg/apis/experimental"
```

Comment:
IIRC, this is cuase by the ThirdPartyResource which is registered in both v1 and experimental.
I will send another PR to remove those from v1.